sdks/python/apache_beam/transforms/core.py

```diff
@@ -742,25 +736,23 @@ def Map(fn_or_label, *args, **kwargs):  # pylint: disable=invalid-name
   # Proxy the type-hint information from the original function to this new
   # wrapped function.
   get_type_hints(wrapper).input_types = get_type_hints(fn).input_types
-  output_hint = get_type_hints(fn).simple_output_type(label)
+  output_hint = get_type_hints(fn).simple_output_type(None)
```

Comment:
Done. simple_output_type uses the label to create a clean error so we should propagate the default label here.